domain_access/domain_access.module

```diff
@@ -212,6 +213,10 @@ function domain_access_form_devel_generate_form_user_alter(&$form, &$form_state,
 
 /**
  * Implements hook_domain_references_alter().
+ *
+ * @param \Drupal\Core\Database\Query\SelectInterface $query
+ * @param \Drupal\user\Entity\User $account
+ * @param array $context
```

Comment:
hook implementation params should not need documenting as per https://www.drupal.org/node/1354#hookimpl